src/components/EventsGeneralInfo.jsx

```diff
@@ -0,0 +1,256 @@
+  const [isEditModalVisible, setIsEditModalVisible] = useState(false);
+
+  const showModalEventType = () => {
+    setIsModalVisible(true);
+  };
+
+  const showModalAddEventType = () => {
+    setIsAddModalVisible(true);
+  };
+
+  const showModalEditEventType = () => {
+    setIsModalVisible(false);
+    setIsEditModalVisible(true);
+  };
+
+  const handleOk = () => {
+    setIsModalVisible(false);
+  };
+
+  const handleCancel = () => {
+    setIsModalVisible(false);
+  };
+
+  const handleAddOk = () => {
+    setIsAddModalVisible(false);
+  };
+
+  const handleAddCancel = () => {
+    setIsAddModalVisible(false);
+  };
+
+  const handleEditOk = () => {
+    setIsEditModalVisible(false);
+  };
+
+  const handleEditCancel = () => {
+    setIsEditModalVisible(false);
+  };
+
+  const volunteerTypeMenu = (
+    <Menu>
+      <Menu.Item>
+        <a target="_blank" rel="noopener noreferrer" href="https://www.antgroup.com">
+          1st menu item
+        </a>
+      </Menu.Item>
+      <Menu.Item>
+        <a target="_blank" rel="noopener noreferrer" href="https://www.aliyun.com">
+          2nd menu item
+        </a>
+      </Menu.Item>
+      <Menu.Item>
+        <a target="_blank" rel="noopener noreferrer" href="https://www.luohanacademy.com">
+          3rd menu item
+        </a>
+      </Menu.Item>
+    </Menu>
+  );
+
+  const tagList = [
+    { name: 'First Aid Training' },
+    { name: 'Can Drive' },
+    { name: 'Age 18 or Older' },
+  ];
+
+  const [list, updateList] = useState(tagList);
+
+  const handleRemoveItem = e => {
+    const name = e.target.getAttribute('name');
+    updateList(list.filter(item => item.name !== name));
+  };
+
+  const [state, setState] = useState('start');
+
+  const addItem = item => {
+    tagList.name = item.target.value;
+  };
+
+  return (
+    <div>
+      <h1> General Information </h1>
+      <Form
+        labelCol={{ span: 6 }}
+        wrapperCol={{ span: 14 }}
+        name="nest-messages"
+        // onFinish={onFinish}
+        // validateMessages={validateMessages}
+        size={componentSize}
+        onValuesChange={onFormLayoutChange}
+      >
+        <Form.Item label="Event Name">
+          <Input placeholder="Ex. Food Running Event" />
+        </Form.Item>
+
+        <Form.Item label="Date / Time">
+          <DatePicker placeholder="Select date" />
+          <TimePicker placeholder="Select time" />
+        </Form.Item>
+
+        <Form.Item label="Event Type">
+          <Button onClick={showModalEventType}>Type</Button>
+          <Button type="link" onClick={showModalAddEventType}>
+            New Event Type
+          </Button>
+          {/* Event Type Pop Up */}
+          <Modal
+            title="Event Type"
+            visible={isModalVisible}
+            onOk={handleOk}
+            onCancel={handleCancel}
+            footer={[
+              <Button key="submit" type="primary" onClick={handleOk}>
```

Comment:
full functionality of New Event Type not done: once a new type is submitted, it should appear in the Event Type dropdown